web-src/src/admin/components/scripts-config/ScriptConfig.vue

```diff
@@ -33,14 +37,20 @@ export default {
   },
 
   methods: {
-    ...mapActions('scriptConfig', ['init', 'save'])
+    ...mapActions('scriptConfig', ['init', 'save', 'deleteScript']),
+    async onDeleteScript() {
+        await this.deleteScript({scriptName: this.scriptName});
+    }
```

Comment:
I think this is redundant, you can just call `deleteScript` from `<PromisableButton :click="onDeleteScript"`

Reply:
when i did that without the await it threw me an error in the console `TypeError: this.click() is undefined`. I could not figure out why, but this way it is not showing up, when the button gets clicked

Reply:
To be honest I don't know how you did it.
I just used the same approach as in "save" button and it worked for me. I pushed this change to your PR

Reply:
Nicely done. If it is doing it's thing without any errors, that's a win